rebar.config

```diff
@@ -14,4 +12,4 @@
                     ]
             }]}.
 
-{erl_opts, [debug_info, warnings_as_errors]}.
+{erl_opts, [debug_info]}. %, warnings_as_errors]}.
```

Comment:
We should add the warnings as error back in.

Reply:
Done.